packages/dotenv/dotenv.0.0.1/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "Jose Nogueira <ze@thatportugueseguy.com>"
+authors: [ "Jose Nogueira <ze@thatportugueseguy.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/thatportugueseguy/ocaml-dotenv"
+dev-repo: "git+https://github.com/thatportugueseguy/ocaml-dotenv.git"
+bug-reports: "https://github.com/thatportugueseguy/ocaml-dotenv/issues"
+doc: "https://thatportugueseguy.github.io/ocaml-dotenv"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.05"}
+  "base"
+  "stdio"
+  "uutf"
+  "dune"
```

Comment:
```suggestion
  "dune" {>= "1.1"}
```
Your `dune-project` file says this project requires at least dune 1.1

Reply:
Thank you, that's righ. Committed.